Cookbook/Technical-Documents/PullAssigners.md

```diff
@@ -0,0 +1,102 @@
+   * If you need a different list to pick your reviewers from, you'd need to create a team via [this page on GitHub](https://github.com/orgs/Babylonpartners/new-team), and once it's created, go to the team's page and add Members to that new team ([e.g. here for `ios-bot-owners`](https://github.com/orgs/Babylonpartners/teams/ios-bot-owners/members))
+
+2. Next, **make sure that this GitHub team –containing the people you want as reviewers– has access to your repository**, via the "Repositories" tab in the team's page ([e.g. here for `ios-bot-owners`](https://github.com/orgs/Babylonpartners/teams/ios-bot-owners/repositories))
+
+### The proxy team to trigger Pull Assigners
+
+1. Now, we will need another, separate, GitHub team –called the "Proxy team" in Pull Assigner's vocabulary– which will be the team that you'll assign to your PRs via `CODEOWNERS` to trigger Pull Assigner on those PRs (†)
+
+   * For the iOS repos, the proxy team we use –to assign people from the `@Babylonpartners/iOS-Admin` to our PRs– is called `@Babylonpartners/iOS-PullAssigner`, so if you are using the same list of people from `iOS-Admin` you can use the `iOS-PullAssigner` proxy team directly without creating a new one
+   * If instead you created a separate team because for your case the list of people in `iOS-Admin` wasn't matching the people you wanted to assign PRs for your repo, you'll need to [create a new GitHub team again](https://github.com/orgs/Babylonpartners/new-team), but this time leave it with no member in it (since it will only act as proxy)
+
+2. Then, again, **make sure that the proxy team has access to your repository**, via the "Repositories" tab in the team's page (e.g. https://github.com/orgs/Babylonpartners/teams/ios-pullassigner/repositories)
+
+> _(†) Side note: This proxy team is not strictly necessary, but if you don't use a proxy team and instead use the team you just declared above directly, that means that everybody will be notified every time a new PR is created and `CODEOWNERS` affects that team containing all your reviewers to pick from, before that team gets removed from the reviewers by Pull Assigners once it has assigned only N people from the team. This is why using a proxy team without any member in it to trigger Pull Assigner is usually preferrable_
+
+### Configure Pull Assigners with those GitHub teams
+
+If you're reusing the `iOS-Admin` and `iOS-PullAssigner` teams for your repo setup, you have nonthing special to configure on https://pullreminders.com as those teams are already configured. You can skip to the next section.
+
+If you've created new teams (team listing reviewers + proxy team) because you need to act on a different list of people for your repo, you'll need to log in to https://pullreminders.com and add the GitHub teams to Pull Assigner's configuration via [this screen](https://pullreminders.com/installs/6124714/assigner)
```

Comment:
```suggestion
But if you have created new teams (team listing reviewers + proxy team) –because you needed to act on a different list of people for your repo– you'll need to log in to https://pullreminders.com and add the GitHub teams to Pull Assigner's configuration via [this screen](https://pullreminders.com/installs/6124714/assigner)
```